ppafm/io.py

```diff
@@ -451,7 +451,8 @@ def loadGeometry(fname=None,format=None,params=None):
         # The automatic generated lattice vector should enclose the whole area filled with atoms as well as the whole scanning area, plus the default padding.
         if np.allclose(lvec[i + 1, :], 0):
             params["PBC"] = False
-            lvec[i + 1, i] = probeMax[i] + pad
+            #lvec[i + 1, i] = probe_max[i] - probe_min[i] + 2*pad
```

Comment:
This comment line will be activated once we deal with a potential non-zero origin properly.